unity-renderer/Assets/Scripts/MainScripts/DCL/Components/UUIDComponent/InteractionHoverCanvas/InteractionHoverCanvasController.cs

```diff
@@ -41,7 +41,8 @@ public void Setup(string button, string feedbackText, IDCLEntity entity)
 
     void ConfigureIcon(string button)
     {
-        hoverIcon?.SetActive(false);
+        if ( hoverIcon != null )
+            hoverIcon.SetActive(false);
```

Comment:
Just as a curiosity: Why did you have to make this change? To use the operator (`?`) is the same as check the null in the conditional, right?

Reply:
On Unity objects, the equality and inequality operators are overloaded. Using the null-conditional operator `?` bypass this checks. So in theory using them on those objects can cause problems. (even Rider gives out a warning about this).

This until now caused no problems, but started to crash when I enabled the "Enter Play Mode Options" checkbox that for some strange reason speeds up the editor by a LOT.
![image](https://user-images.githubusercontent.com/6875814/124780899-58af6c00-df19-11eb-97c5-69777c3c3eb4.png)

Reply:
Good to know! Thanks!